### ISSUE TITLE
Small fee should be applied for oracle transactions

### ISSUE THREAD

**Issue by robert-zaremba** [umee-network/umee#1214]
## Summary

Add static fee limit for price transactions.

Nice to have for v3, can be pushed back to v4.

## Problem Definition

Today, the oracle price transactions can be provided for free as long as the gas per message is in the `MaxOracleMsgGasUsage` limit.

## Proposal
For keeping the process more sustainable we need to add additional fee mechanism, with a lower limit than https://github.com/umee-network/umee/issues/1017
Might be 0.001umee per message. For reference, bank send tx cost about 0.005-0.01umee, and price feed transaction is a bit more resource intensive.

---

## For Admin Use

- [ ] Not duplicate issue
- [ ] Appropriate labels applied
- [ ] Appropriate contributors tagged
- [ ] Contributor assigned/self-assigned

**Comment by robert-zaremba** [comment]
Thinking more about it... Even if we enforce a small fee then this won't prevent accessing a state for free (because this is happening in CheckTx phase - so before we can charge any fee).
So my conclusion is to keep oracle tx for free (with current rules: checking gas limit and tx is only oracle messages) and optimize the CheckTx by using cache (added new issue for that: https://github.com/umee-network/umee/issues/1327) to prevent state access.
